social/backends/twitter.py

```diff
@@ -27,7 +27,7 @@ def get_user_details(self, response):
         """Return user details from Twitter account"""
         fullname, first_name, last_name = self.get_user_names(response['name'])
         return {'username': response['screen_name'],
-                'email': '',  # not supplied
+                'email': response.get('email'),
```

Comment:
I think it should be `response.get('email','')` for backwards compatibility (empty string and not None when email is missing).